### ISSUE TITLE
Broken repository url with amdgpu-install 5.5.50500-1 for Ubuntu

### ISSUE THREAD

**Issue by illwieckz** [ROCm/ROCm#2111]
When installing `amdgpu-install_5.5.50500-1_all.deb` it writes a file named `/etc/apt/sources.list.d/amdgpu-proprietary.list` that contains this content:

```
# Enabling this repository requires acceptance of the following license:
# /usr/share/amdgpu-install/AMDGPUPROEULA
#deb https://repo.radeon.com/amdgpu/@AMDGPUVER@/ubuntu jammy proprietary
```

Notice the bad `@AMDGPUVER@` string (instead of `5.5` version string) that makes this file unusable.

**Comment by illwieckz** [comment]
Yes, I had to set `5.4.5` instead:

```
# Enabling this repository requires acceptance of the following license:
# /usr/share/amdgpu-install/AMDGPUPROEULA
deb https://repo.radeon.com/amdgpu/5.4.5/ubuntu jammy proprietary
```

**Comment by dictvm** [comment]
Stumbled upon this as well. Am I mistaken if I assume that having fixed versions (like 5.6) in my sources.list-files would mean that ROCM won't update to versions (like 5.7) in the future? That'd be my expectation and I don't really want to manually look up ROCM versions every now an then.

**Comment by illwieckz** [comment]
My experience is that when updating the package, the version string in the `amdgpu-proprietary.list` file gets replaced by `@AMDGPUVER@` again.

**Comment by dictvm** [comment]
Thanks to the both of you! I'll keep it as it is for now. 5.6 is performing decently and I'll know if versions >5.6  will be released so I can manually update anyways!

**Comment by ppanchad-amd** [comment]
@illwieckz Apologies for the lack of response. Can you please test with latest ROCm 6.1.1? If resolved, please close ticket. Thanks!

**Comment by illwieckz** [comment]
@ppanchad-amd Hi, the problem is fixed since ROCm 5.7.
